app/services/pii/attribute.rb

```diff
@@ -0,0 +1,25 @@
+module Pii
+  class Attribute
+    attr_accessor :user, :norm
+
+    def initialize(user: nil, norm: nil)
+      @user = user
+      @norm = norm
+    end
+
+    def method_missing(method_sym, *arguments, &block)
+      user.send method_sym, *arguments
+    end
+
+    def respond_to_missing?(method_sym, include_private)
+      user.respond_to?(method_sym, include_private)
+    end
+
+    delegate :blank?, :present?, :to_s, :to_date, :==, :eql?, to: :user
```

Comment:
This basically creates backwards-compatability between a `String` and a `Pii::Attribute`. Notice how few tests need to change.